packages/jest/src/migrations/update-15-8-0/update-configs-jest-29.ts

```diff
@@ -10,21 +10,37 @@ import { tsquery } from '@phenomnomnominal/tsquery';
 import * as ts from 'typescript';
 import { JestExecutorOptions } from '../../executors/jest/schema';
 import { forEachExecutorOptionsInGraph } from '@nrwl/workspace/src/utilities/executor-options-utils';
+import { findRootJestPreset } from '../../utils/config/find-root-jest-files';
 
 export async function updateConfigsJest29(tree: Tree) {
+  const rootPreset = findRootJestPreset(tree);
+  let updateSnapshotConfig = false;
+  if (rootPreset && tree.exists(rootPreset)) {
```

Comment:
Move this to the bottom so we can get the target names

Reply:
fixed!